apps/backend/src/build-notification/comment.ts

```diff
@@ -44,45 +44,45 @@ export const getCommentBody = async (props: {
 
   const hasMultipleProjects =
     new Set(builds.map((build) => build.projectId)).size > 1;
-  const aggregateStatuses = await Build.getAggregatedBuildStatuses(builds);
-  const buildRows = await Promise.all(
-    builds
-      // Filter out builds that are not associated with a project that has PR comments enabled.
-      // We don't filter at SQL level because we still want to know if it's linked to multiple projects
-      // to determine if we should prepend the project name to the build name.
-      .filter((build) => {
-        invariant(build.project, "Relation `project` should be fetched");
-        return build.project.prCommentEnabled;
-      })
-      .map(async (build, index) => {
-        invariant(build.project, "Relation `project` should be fetched");
+  const [aggregateStatuses, urls] = await Promise.all([
+    Build.getAggregatedBuildStatuses(builds),
+    Promise.all(builds.map((build) => build.getUrl())),
```

Comment:
Why do you need a second `Promise.all` ?

Reply:
So I can get the urls in an array.